cli/commands/search.py

```diff
@@ -0,0 +1,166 @@
+import rich
+import json
+import chevron
+from .. import ContentType, term_directory
+from . import (
+    prompt_user_to_choose_a_content_type,
+    session,
+    rank_client,
+    prompt_user_to_choose_a_remote_index,
+    prompt_user_to_choose_a_local_query,
+)
+import typer
+from typing import Optional
+from datetime import datetime
+from ..services import elasticsearch
+
+
+app = typer.Typer(
+    name="search",
+    help="Run searches against candidate indices with candidate queries",
```

Comment:
I think this description needs to be tweaked - afaict there is nothing in the docs which will tell me what will actually happen in terms of side effects or terminal output if I run this command.